### PR TITLE
Troelsen

### PR DESCRIPTION
**Pull request by LitvinovichPavel**

### CODE REVIEW

**Comment by TRENLitvinovich** [comment]
Hello!

File: Troelsen_static_class_/Troelsen_static_class_/Program.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+namespace Troelsen_static_class
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            TimeUtilClass.PrintDate();
+            TimeUtilClass.PrintTime();
+
```
Comment (TRENLitvinovich) [inline Troelsen_static_class_/Troelsen_static_class_/Program.cs]: Delete

**Comment by LitvinovichPavel** [comment]
Hello

File: Troelsen_static_class_/Troelsen_static_class_.sln [hunk 2]
```diff
@@ -1,25 +1,25 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
-VisualStudioVersion = 16.0.29519.87
+VisualStudioVersion = 16.0.29613.14
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "lecture_08", "lecture_08\lecture_08.csproj", "{3FF48E0E-099A-4F97-8A90-9033F1CA633C}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Troelsen_static_class_", "Troelsen_static_class_\Troelsen_static_class_.csproj", "{9C8A9D2F-8E10-4ABD-847C-B23E8899B720}"
 EndProject
 Global
```
Comment (TRENLitvinovich) [inline Troelsen_static_class_/Troelsen_static_class_.sln]: Good

File: Troelsen_encapsulation.02/Troelsen_encapsulation.02/Employee.cs [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        public int Age
+        {
+            get => empAge;
+            set => empAge = value;
+        }
+        public float Pay
+        {
+            get => cuppPay;
+            set => cuppPay = value;
+        }
```
Comment (TRENLitvinovich) [inline Troelsen_encapsulation.02/Troelsen_encapsulation.02/Employee.cs]: property

File: Troelsen_encapsulation.02/Troelsen_encapsulation.02/Program.cs [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+        static void Main(string[] args)
+        {
+            Employee emp = new Employee("TOOOM", 2414, 30, 30000);
+            emp.DisplayStatus();
+
+            emp.Name = "AAA4XI";
+            emp.ID = 9379992;
+            emp.Age = 31;
+            emp.Pay = 33333;
+            emp.DisplayStatus();
```
Comment (TRENLitvinovich) [inline Troelsen_encapsulation.02/Troelsen_encapsulation.02/Program.cs]: Bla bla bla

**Pull request closed without merging**